server/configuration/accounts_meld.coffee

```diff
@@ -17,6 +17,10 @@ Accounts.updateOrCreateUserFromExternalService = (serviceName, serviceData, opti
 	if serviceName is 'linkedin'
 		serviceData.email = serviceData.emailAddress
 
+	if serviceName is 'sandstorm'
+		console.log "Grab data from: ", serviceData
+		serviceData.username = serviceData.preferredHandle
```

Comment:
I think this will fail if the handle is already taken in this grain.

There is already some existing code which initializes the username based on `profile.name` (which Sandstorm sets to the user's real name, like "Kenton Varda"). Would it be easy to change that code to use `preferredHandle` instead?